Scripts/capture-build-details.sh

```diff
@@ -20,3 +21,11 @@ plutil -replace com-loopkit-Loop-srcroot -string "${SRCROOT}" "${plist}"
 plutil -replace com-loopkit-Loop-build-date -string "$(date)" "${plist}"
 plutil -replace com-loopkit-Loop-xcode-version -string "${XCODE_PRODUCT_BUILD_VERSION}" "${plist}"
 
+profile_path="${HOME}/Library/MobileDevice/Provisioning Profiles/${EXPANDED_PROVISIONING_PROFILE}.mobileprovision"
```

Comment:
[nit] Choose either all upper or all lower case environment variable names?

Reply:
all local vars are lowercase.

Reply:
Ah, okay. Is that a Loop standard we should be using? I think the scripts that I wrote or updated used all caps for all environment variables (global or local).